docs/general/teams/qa-team.md

```diff
@@ -0,0 +1,34 @@
+---
+title: QA Responsibilities
```

Comment:
Title should probably match the name of the link: "Quality Assurance". Recommend also renaming the file to _quality-assurance.md_.

Reply:
Agreed, this was a relic of a previous iteration.  Updated.